components/vr-audio.js

```diff
@@ -0,0 +1,46 @@
+/* globals define */
+(function(define){'use strict';define(function(require,exports,module){
+
+  document.registerElement(
+    'vr-audio',
+    {
+      prototype: Object.create(
+        VRObject.prototype, {
+          init: {
+            value: function() {
+              var src = this.getAttribute('src');
+              var loop = this.hasAttribute('loop');
+              var xPos = +this.getAttribute('x') || 0;

```

Comment:
you inherit from VRObject the updates to x,y,z rotX, rotY, rotZ are taken care for you. You can remove: 
```javascript
var xPos = +this.getAttribute('x') || 0;
 var yPos = +this.getAttribute('y') || 0;
var zPos = +this.getAttribute('z') || 0;
```
and 
```javascript
sound.position.set(xPos, yPos, zPos);
```